src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (Object.prototype.hasOwnProperty.call(invertedRobot, robot[key])) {
```

Comment:
Using `Object.prototype.hasOwnProperty.call` is a safe way to check for property existence. However, since you are creating a new object `invertedRobot` which does not inherit from anything, you can use the simpler `invertedRobot.hasOwnProperty(robot[key])` or even `robot[key] in invertedRobot` for readability.